### ISSUE TITLE
[Feature] Check / Cleanup disk space before pull

### ISSUE THREAD

**Issue by NorseGaud** [chef/anka-buildkite-plugin#12]
I'm trying to get anka folks to add this as behavior to anka pull, but if they tell me no, we'll need to add it to the plugin. My disk space is limited on the mac minis I use for anka and I can only store 2 VM templates locally. If I add a third in the near future, I'll end up running out of disk space. Anka pulling should do a disk space check and delete one unused template before doing the actual download.

I'll reply in a few days with what they say.

**Comment by NorseGaud** [comment]
> We have this feature when using from the controller apis, we will do internal thinking about how something in the anka command line can help in cases that VMs aren't started from the controller. 
> Regards,
> Izik (Veertu Inc)

**Comment by tduffield** [comment]
Is this still an issue?

**Comment by NorseGaud** [comment]
Definitely! We should check disk space and if it’s close to full, do a registry -s to shrink the tags on the node. Shouldn’t take too long to implement.

**Comment by tduffield** [comment]
Does this command exit non-zero if there is insufficient space, or do we need to parse the output of this command?
